integration/test-integration/src/test/resources/testng.xml

```diff
@@ -44,6 +44,7 @@
             <class name="org.wso2.choreo.connect.tests.testCases.jwtValidator.InternalKeyTestCase"/>
             <class name="org.wso2.choreo.connect.tests.testCases.jwtValidator.CustomAuthHeaderTestCase"/>
             <class name="org.wso2.choreo.connect.tests.testCases.apiDeploy.QSGTestCase"/>
+            <class name="org.wso2.choreo.connect.tests.testCases.vhost.VhostAPICtlTestCase"/>
```

Comment:
Can we add vhsot test case when API created with vhost from APIM as well

Reply:
Will be addressed with new PR